tests/Operation/FindFunctionalTest.php

```diff
@@ -123,6 +125,45 @@ public function provideTypeMapOptionsAndExpectedDocuments()
         ];
     }
 
+    public function testMaxAwaitTimeMS()
+    {
+        $maxAwaitTimeMS = 10;
+
+        // Create a capped collection.
+        $databaseName = $this->getDatabaseName();
+        $cappedCollectionName = $this->getCollectionName() . '.capped';
+        $cappedCollectionOptions = [
+            'capped' => true,
+            'max' => 100,
+            'size' => 1048576,
+        ];
+
+        $operation = new CreateCollection($databaseName, $cappedCollectionName, $cappedCollectionOptions);
+        $operation->execute($this->getPrimaryServer());
+
+        // Insert documents into the capped collection.
+        $cappedNamespace = $databaseName . '.' . $cappedCollectionName;
+        $bulkWrite = new BulkWrite(['ordered' => true]);
+        $bulkWrite->insert(['_id' => 1]);
+        $result = $this->manager->executeBulkWrite($cappedNamespace, $bulkWrite);
+
+        // Get a cursor of type TAILABLE_AWAIT.
+        $operation = new Find($databaseName, $cappedCollectionName, [], ['cursorType' => 3, 'maxAwaitTimeMS' => $maxAwaitTimeMS]);
```

Comment:
`3` can be replaced with `Find::TAILABLE_AWAIT`. Not that I expect the constants to change, but this avoids a [magic number](https://stackoverflow.com/q/47882/162228) and sets a good example for any users that might be reading through our tests for an example.